src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -369,17 +369,18 @@ internal virtual SmiExtendedMetaData[] GetInternalSmiMetaData()
                             length /= ADP.CharSize;
                         }
 
-                        metaDataReturn[returnIndex] = new SmiQueryMetaData(
+                        metaDataReturn[returnIndex] =
+                            new SmiQueryMetaData(
                                                         colMetaData.type,
                                                         length,
                                                         colMetaData.precision,
                                                         colMetaData.scale,
                                                         (null != collation) ? collation.LCID : _defaultLCID,
                                                         (null != collation) ? collation.SqlCompareOptions : SqlCompareOptions.None,
                                                         colMetaData.udt?.Type,
-                                                        false,  // isMultiValued
```

Comment:
Actually instead of changing the indent, could you use the named parameters here? 
Tbh, I kinda forget the rules for using those, ie, can they be omitted in later arguments or once they start do they have to be used until the end?